src/main/java/core/basesyntax/model/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.model;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Colors[] colors = Colors.values();
```

Comment:
lets define colors as constant, you can read about it [here](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)